java/timeouts_application/src/main/java/org/demo/util/Commons.java

```diff
@@ -0,0 +1,32 @@
+package org.demo.util;
+
+import com.netflix.conductor.common.metadata.tasks.TaskDef;
+import com.netflix.conductor.common.metadata.tasks.TaskType;
+import com.netflix.conductor.common.metadata.workflow.WorkflowDef;
+import com.netflix.conductor.common.metadata.workflow.WorkflowTask;
+
+import java.util.Arrays;
+import java.util.Map;
+
+public class Commons {
```

Comment:
Simpler if this a parent class that all the logic classes inherit from. This class would supply the OrkesWorkflowClient and OrkesMetadataClient directly, as well as GetUserTask() and GetWorkflowDef() to avoid having util classes and static calls.

Reply:
We want to keep the implementations separate to allow us to change the workflow definitions and task definitions at will and not deeply coupled to the individual case implementation.